contentctl/objects/unit_test_result.py

```diff
@@ -54,6 +54,15 @@ def get_summary_dict(
 
         return results_dict
 
+    def set_manual_test(self,manual_test_text:str):
```

Comment:
I think marking this as a success is OK for now. But, my integration testing changes introduce a notion of states (PASS, FAIL, ERROR, SKIP) and I think SKIP would be more appropriate for the manual_tests ultimately

Reply:
The code has now been updated to use the SKIP state. This carries through to Integration Testing as well!